service/controller/resource/pvc/local_pvc.go

```diff
@@ -0,0 +1,63 @@
+package pvc
+
+import (
+	"github.com/giantswarm/apiextensions/v3/pkg/apis/provider/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/kvm-operator/service/controller/key"
+)
+
+const (
+	LabelMountTag = "mount-tag"
+)
+
+func newLocalPVCs(customObject v1alpha1.KVMConfig, pvsList *corev1.PersistentVolumeList) []*corev1.PersistentVolumeClaim {
+	var persistentVolumeClaims []*corev1.PersistentVolumeClaim
+
+	for i, workerKVM := range customObject.Spec.KVM.Workers {
+		workerNode := customObject.Spec.Cluster.Workers[i]
+
+		for _, hostVolume := range workerKVM.HostVolumes {
+			for _, pv := range pvsList.Items {
+				if pv.ObjectMeta.Labels[LabelMountTag] == hostVolume.MountTag {
+					persistentVolumeClaim := &corev1.PersistentVolumeClaim{
+						TypeMeta: metav1.TypeMeta{
+							Kind:       "PersistentVolumeClaim",
+							APIVersion: "v1",
+						},
+						ObjectMeta: metav1.ObjectMeta{
+							Name: key.LocalWorkerPVCName(key.ClusterID(customObject), key.VMNumber(i)),
+							Labels: map[string]string{
+								"app":      key.WorkerID,
+								"cluster":  key.ClusterID(customObject),
+								"customer": key.ClusterCustomer(customObject),
+								"node":     workerNode.ID,
+							},
+							Annotations: map[string]string{
+								"volume.beta.kubernetes.io/storage-class": LocalStorageClass,
```

Comment:
I think you should use `storageClassName` instead of this annotation. According to https://kubernetes.io/docs/concepts/storage/persistent-volumes/#class-1, `In the past, the annotation volume.beta.kubernetes.io/storage-class was used instead of storageClassName attribute. This annotation is still working; however, it won't be supported in a future Kubernetes release.`

Reply:
oh makes a lot of sense. did not check that. then for sure let's change it.